src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -330,8 +330,8 @@ int RadosBucket::remove(const DoutPrefixProvider* dpp,
   RGWObjVersionTracker ot;
 
   // if we deleted children above we will force delete, as any that
-  // remain is detritus from a prior bug
-  ret = store->getRados()->delete_bucket(info, ot, y, dpp, !delete_children);
+  // remain is detrius from a prior bug
```

Comment:
detritus is the correct spelling. this was a conflict from https://github.com/ceph/ceph/pull/51229

Reply:
bad rebase